wrapping/python/apps/assemble.py

```diff
@@ -0,0 +1,289 @@
+import sys
+import argparse
+#import openmeeg as om
+
+def parser(argv):
```

Comment:
Rather than a single monolithic file I find MNE-Python's way of doing things much easier -- each `mne whatever` has a `mne/commands/mne_whatever.py` that you modify with your own arguments

https://github.com/mne-tools/mne-python/blob/main/mne/commands/mne_browse_raw.py

I think it's more maintainable than this approach, where everything lives in a single namespace

I can convert it to this format if you want

Reply:
I'm not sure I understand. each `openmeeg command` already has its own `command.py` file. If you meanto make the same at each subcommand level i.e. `openmeeg command subcommand` into its own  `subcommand.py` file, then I totally agree. This will be particularly useful for the assemble and gain commands, which are huge.

What I just want to have (but I'm sure that is also done with mne) is to make sure that all:
```
openmeeg --help
openmeeg command --help
openmeeeg command subcommand --help
```
provide the relevant messages.

Note though that there are two commands in assemble that share the same `processing`, but that's largely doable with your proposed split. Also, I like better the run() name, so if we split at the subcommand level, there is no need to call each function with its own name, they can all have the same run method.

Did I understood your proposal correctly ?
If yes, be my guest, as I have other tasks to do for the next hours/days...
In particular, if you have some proper install example to follow from mne that would be great too (coming commit, but probably with wrong install places).

PS: I did not knew that mne-python had a command line interface !!! There is always something new to learn...

Reply:
> I'm not sure I understand. each openmeeg command already has its own command.py file.

Ahh I just misread / didn't look closely enough, I didn't realize there were `subcommand`s

> In particular, if you have some proper install example to follow from mne that would be great too (coming commit, but probably with wrong install places).

Usually with `setuptools` at least you provide an `entry_point`:

https://github.com/mne-tools/mne-python/blob/7730df72dc51f663fe557e997b6f5486111f3e43/setup.py#L138-L139

This should get picked up automatically when we build wheels, and it's easy to add to `conda` when we package there. I'm not sure how to do it with `cmake` actually, but hopefully it's not too hard there either...